### DISCUSSION TITLE
Current month filter

### DISCUSSION THREAD

**Discussion by nordcart** [directus/directus#14963] [Archive]
Hi!

Is it possible to retrieve current month in a number-format and use it in a filter? For example it is now august-->8

Can this number be adjusted like: $NOW(<adjustment>) ?

**Comment by rijkvanzanten** [comment]
There's currently no way to achieve that. That being said, supporting `$NOW()` _inside_ the existing `month()` function sounds like a great feature request 🙂 Hopefully in the future, you could do something like `month($NOW)`

**Comment by nervusdm** [comment]
As a workaround, I guess that cron job can make it.
(each start of month.., you replace the last "2022-07-01" by ...)
